data_structures/node_based_graph.hpp

```diff
@@ -60,6 +60,7 @@ struct NodeBasedEdgeData
     bool access_restricted : 1;
     bool reversed : 1;
     bool roundabout : 1;
+    bool startpoint : 1;
     TravelMode travel_mode : 4;
```

Comment:
Is this struct defined at multiple locations? Would prefer to move it to an include and only define it once.